os-impl-no-symtab.c out of date

**Describe the bug**
Conflicting types for the implementations

**To Reproduce**
Attempt to compile w/ os-impl-no-symtab.c:  OSAL_CONFIG_INCLUDE_DYNAMIC_LOADER FALSE

**Expected behavior**
Should compile

**Code snips**
Many of the implementations don't match, sample:
```
src/os/portable/os-impl-no-symtab.c:55:7: error: conflicting types for ‘OS_ModuleSymbolLookup_Impl’
 int32 OS_ModuleSymbolLookup_Impl(uint32 local_id, cpuaddr *SymbolAddress, const char *SymbolName)
src/os/shared/inc/os-shared-module.h:115:7: note: previous declaration of ‘OS_ModuleSymbolLookup_Impl’ was here
 int32 OS_ModuleSymbolLookup_Impl(const OS_object_token_t *token, cpuaddr *SymbolAddress, const char *SymbolName);

```

**System observed on:**
 - Hardware: cFS Dev Server
 - OS: Ubuntu 18.04
 - Versions cFS Bundle

**Additional context**
Seen when adding a coverage test for this code (#795), #796

**Reporter Info**
Jacob Hageman - NASA/GSFC

--- a/src/os/portable/os-impl-no-symtab.c
+++ b/src/os/portable/os-impl-no-symtab.c
@@ -31,43 +31,34 @@
 #include "os-shared-module.h"
 
 /*----------------------------------------------------------------
+ * Implementation for no dynamic loader configuration
  *
- * Function: OS_GlobalSymbolLookup_Impl
- *
- *  Purpose: Implemented per internal OSAL API
- *           See prototype for argument/return detail
- *
+ * See prototype for argument/return detail
  *-----------------------------------------------------------------*/
 int32 OS_GlobalSymbolLookup_Impl(cpuaddr *SymbolAddress, const char *SymbolName)
 {
     return OS_ERR_NOT_IMPLEMENTED;
 
-} /* end OS_GlobalSymbolLookup_Impl */
+}
 
 /*----------------------------------------------------------------
+ * Implementation for no dynamic loader configuration
  *
- * Function: OS_ModuleSymbolLookup_Impl
- *
- *  Purpose: Implemented per internal OSAL API
- *           See prototype for argument/return detail
- *
+ * See prototype for argument/return detail
  *-----------------------------------------------------------------*/
-int32 OS_ModuleSymbolLookup_Impl(uint32 local_id, cpuaddr *SymbolAddress, const char *SymbolName)
+int32 OS_ModuleSymbolLookup_Impl(const OS_object_token_t *token, cpuaddr *SymbolAddress, const char *SymbolName)
 {
     return OS_ERR_NOT_IMPLEMENTED;
 
-} /* end OS_ModuleSymbolLookup_Impl */
+}
 
 /*----------------------------------------------------------------
+ * Implementation for no dynamic loader configuration
  *
- * Function: OS_SymbolTableDump_Impl
- *
- *  Purpose: Implemented per internal OSAL API
- *           See prototype for argument/return detail
- *
+ * See prototype for argument/return detail
  *-----------------------------------------------------------------*/
 int32 OS_SymbolTableDump_Impl(const char *filename, size_t SizeLimit)
 {
     return (OS_ERR_NOT_IMPLEMENTED);
 
-} /* end OS_SymbolTableDump_Impl */
+}

